packages/devtools_app/lib/src/screens/inspector_v2/inspector_tree_controller.dart

```diff
@@ -990,11 +1045,6 @@ class _InspectorTreeState extends State<InspectorTree>
     treeController?.addClient(this);
   }
 
-  @override
-  void onChanged() {
-    setState(() {});
-  }
-
   @override
```

Comment:
this can be done in a follow up or cleanup after the V2 feature is complete. But it could be nice to refactor all the inspector code to follow patterns that other areas of the Devtools codebase use. Doing things like keeping 'controller' classes and 'view' classes in separate libraries.

Reply:
Yes! Was planning to do that eventually after realizing how many different classes were in a single file. Opened https://github.com/flutter/devtools/issues/7991 to track